etl-pipeline/processes/record_deleter.py

```diff
@@ -0,0 +1,102 @@
+import os
+from sqlalchemy import joinedload
+
+from logger import create_log
+from model import Edition, Item, Record, Work
+from managers import DBManager, ElasticsearchManager, S3Manager
+
+logger = create_log(__name__)
+
+
+class RecordDeleter:
+
+    def __init__(self, db_manager: DBManager, store_manager: S3Manager, es_manager: ElasticsearchManager):
+        self.db_manager = db_manager
+        self.store_manager = store_manager
+        self.es_manager = es_manager
+
+    def delete_record(self, record: Record):
+        self._delete_record_digital_assets(record)
+        self._delete_record(record)
+
+        logger.info(f'Deleted {record}')
+
+    def _delete_record(self, record: Record):
```

Comment:
The names of both methods are nearly identical here. Is that intentional?

Reply:
I think a different name for the second delete record could be delete_whole_record

Reply:
will separate it out!